docs/umd_bundles_compatibility.md

```diff
@@ -1,20 +1,14 @@
-# Compatibility with legacy UMD bundles
+When adding ILC to the legacy website, you may have an issue when the code that is loaded via regular `<script>` tags works incorrectly.
```

Comment:
I would add header to a doc

Reply:
@b1ff Do you want this heading to be in the markdown itself for the sake of completeness, or to be shown in the documentation?

In terms of the documentation, this heading is useless because MkDocs renders the proper heading anyway:

<img width="578" alt="heading" src="https://user-images.githubusercontent.com/16765690/161712722-834d0127-6f94-4a66-ab80-c888ca4921ec.png">